### PR TITLE
1.6.0

### PR DESCRIPTION
**Pull request by sraghunandan**

### CODE REVIEW

File: content/configuration-parameters.html [hunk 1]
```diff
@@ -624,6 +648,11 @@ <h2>
...
 <td>Each CarbonData file has a companion CarbonIndex file which maintains the metadata about the data. These CarbonIndex files are read and loaded into driver and is used subsequently for pruning of data during queries. These CarbonIndex files are very small in size(few KB) and are many. Reading many small files from HDFS is not efficient and leads to slow IO performance. Hence these CarbonIndex files belonging to a segment can be combined into  a single file and read once there by increasing the IO throughput. This configuration enables to merge all the CarbonIndex files into a single MergeIndex file upon data loading completion.<strong>NOTE:</strong> Reading a single big file is more efficient in HDFS and IO throughput is very high. Due to this the time needed to load the index files into memory when query is received for the first time on that table is significantly reduced and there by significantly reduces the delay in serving the first query.</td>
 </tr>
+<tr>
+<td>carbon.enable.range.compaction</td>
+<td>true</td>
+<td>To configure Ranges-based Compaction to be used or not for RANGE_COLUMN. If true after compaction also the data would be present in ranges.</td>
```
Comment (chetandb) [inline content/configuration-parameters.html:654]: Change Ranges to Range in "To configure Ranges-based Compaction" and "the data would be present in ranges"

File: content/configuration-parameters.html [hunk 2]
```diff
@@ -540,6 +554,16 @@ <h2>
...
 <td>CarbonData will write the min max values for string/varchar types column using the byte count specified by this configuration. Max value is 1000 bytes(500 characters) and Min value is 10 bytes(5 characters). <strong>NOTE:</strong> This property is useful for reducing the store size thereby improving the query performance but can lead to query degradation if value is not configured properly.</td>
 </tr>
+<tr>
+<td>carbon.merge.index.failure.throw.exception</td>
+<td>true</td>
+<td>It is used to configure whether or not merge index failure should result in data load failure also.</td>
+</tr>
+<tr>
+<td>carbon.binary.decoder</td>
+<td>None</td>
```
Comment (chetandb) [inline content/configuration-parameters.html:564]: Default value should be base64 or hex. Cant be None

File: content/dml-of-carbondata.html [hunk 3]
```diff
@@ -578,6 +593,14 @@ <h5>
 <li>If both GLOBAL_SORT_PARTITIONS and SCALE_FACTOR are used at the same time, only GLOBAL_SORT_PARTITIONS is valid.</li>
 <li>The compaction on RANGE_COLUMN will use LOCAL_SORT by default.</li>
 </ul>
+<ul>
+<li>
+<h5>
+<a id="carbon_enable_range_compaction" class="anchor" href="#carbon_enable_range_compaction" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>CARBON_ENABLE_RANGE_COMPACTION</h5>
+</li>
+</ul>
+<p>To configure Ranges-based Compaction to be used or not for RANGE_COLUMN.
```
Comment (chetandb) [inline content/dml-of-carbondata.html:602]: Change Ranges to Range in "To configure Ranges-based Compaction"

File: src/main/webapp/configuration-parameters.html [hunk 4]
```diff
@@ -624,6 +648,11 @@ <h2>
...
 <td>Each CarbonData file has a companion CarbonIndex file which maintains the metadata about the data. These CarbonIndex files are read and loaded into driver and is used subsequently for pruning of data during queries. These CarbonIndex files are very small in size(few KB) and are many. Reading many small files from HDFS is not efficient and leads to slow IO performance. Hence these CarbonIndex files belonging to a segment can be combined into  a single file and read once there by increasing the IO throughput. This configuration enables to merge all the CarbonIndex files into a single MergeIndex file upon data loading completion.<strong>NOTE:</strong> Reading a single big file is more efficient in HDFS and IO throughput is very high. Due to this the time needed to load the index files into memory when query is received for the first time on that table is significantly reduced and there by significantly reduces the delay in serving the first query.</td>
 </tr>
+<tr>
+<td>carbon.enable.range.compaction</td>
+<td>true</td>
+<td>To configure Ranges-based Compaction to be used or not for RANGE_COLUMN. If true after compaction also the data would be present in ranges.</td>
```
Comment (chetandb) [inline src/main/webapp/configuration-parameters.html:654]: Change ranges to range in "To configure Ranges-based Compaction " and "the data would be present in ranges"

File: src/main/webapp/dml-of-carbondata.html [hunk 5]
```diff
@@ -578,6 +593,14 @@ <h5>
 <li>If both GLOBAL_SORT_PARTITIONS and SCALE_FACTOR are used at the same time, only GLOBAL_SORT_PARTITIONS is valid.</li>
 <li>The compaction on RANGE_COLUMN will use LOCAL_SORT by default.</li>
 </ul>
+<ul>
+<li>
+<h5>
+<a id="carbon_enable_range_compaction" class="anchor" href="#carbon_enable_range_compaction" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>CARBON_ENABLE_RANGE_COMPACTION</h5>
+</li>
+</ul>
+<p>To configure Ranges-based Compaction to be used or not for RANGE_COLUMN.
```
Comment (chetandb) [inline src/main/webapp/dml-of-carbondata.html:602]: Change ranges to range in "To configure Ranges-based Compaction"

**Comment by sraghunandan** [comment]
Merged to asf-site

**Pull request closed without merging**